recipe/build.sh

```diff
@@ -1,2 +0,0 @@
-"${PYTHON}" setup.py install --single-version-externally-managed --record=record.txt
-"${PREFIX}/bin/jupyter-nbextension" install --sys-prefix --overwrite --py nbgrader
```

Comment:
Removed the build files and added the simpler `script` line.
The second command and be in the post/pre hooks.